projects/packages/my-jetpack/_inc/components/product-card/stories/index.jsx

```diff
@@ -10,6 +11,9 @@ import React from 'react';
 import ProductCard, { PRODUCT_STATUSES } from '../index.jsx';
 import { initStore } from '../../../state/store';
 
+// Set myJetpackRest global var.
+window.myJetpackRest = {};
```

Comment:
Thinking if we could not have some global decorator to inject this. 🤔  ... But, we can handle this in upcoming PRs

Reply:
Basic because `ProductCardsSection` needs the same thing.

Reply:
yeah, I think it's something to consider, too.